core/server/src/main/java/alluxio/worker/block/BlockMetadataManagerView.java

```diff
@@ -229,4 +229,15 @@ public BlockMeta getBlockMeta(long blockId) throws BlockDoesNotExistException {
       return null;
     }
   }
+
+  /**
+   * Return the byte size of the block.
+   *
+   * @param blockId id of the block
+   * @return block size
+   * @throws BlockDoesNotExistException if no {@link BlockMeta} for this block id is found
+   */
+  public long getBlockSizeIfExist(long blockId) throws BlockDoesNotExistException {
```

Comment:
@Yours-shupeng If you change the signature of `onRemove` from `long blockId` to `BlockMeta meta`, then you do not need this call.